spec/services/reference_status_spec.rb

```diff
@@ -154,4 +154,16 @@
       expect(status.needs_a_replacement_reference?).to be(true)
     end
   end
+
+  describe '#references not requested yet?' do
+    it 'returns true if references not yet requested' do
+      application_form = create(:application_form)
+      create(:reference, :not_requested_yet, application_form: application_form)
+      create(:reference, :not_requested_yet, application_form: application_form)
+
```

Comment:
reference1 = create(:reference, :not_requested_yet, application_form: application_form)
reference2 = create(:reference, :feedback_provided, application_form: application_form)

Reply:
@davidgisbey I have made the suggested changes from both comments. Please review :)

Reply:
Rubocop did not enjoy the second variable being in the test as it claims it is not used. I've edited the test as shown above. This should still produced the desired effect.